pivotal-import/README.md

```diff
@@ -16,8 +16,9 @@ In order to run this, you will require a Pivotal account and the ability to sign
    - Follow instructions printed to the console to ensure the mapping of Pivotal and Shortcut data is complete and correct.
    - Refer to `data/priorities.csv`, `data/states.csv`, and `data/users.csv` to review these mappings.
 1. If the dry-run output looks correct, you can apply the import to your Shortcut workspace by running `make import-apply`
-1. Refer to `data/shortcut_imported_entities.csv` to review all epics, stories, etc. imported successfully into Shortcut.
-   1. If you find that you need to adjust your configuration or your Pivotal data and try again, you can run `make delete` to review a dry-run and `make delete-apply` to actually delete the imported Shortcut epics and stories listed in that CSV file.
+   - The console should print a link to an import-specific Shortcut label page that you can review to find all imported Stories and Epics.
+   - If you run the importer multiple times, you can review all imported Stories and Epics by visiting Settings > Labels and then searching for the `pivotal->shortcut` label and clicking on it.
```

Comment:
@cdemwell Great points! I need to document an explicit set of re-run steps, since the importer currently does not attempt any idempotency across runs; each import creates the full set of stories/epics/etc. over again regardless of what's present in the target Shortcut workspace.

As you mention, I'll queue that up in a subsequent PR. Thanks for the review!